src/components/Nav.tsx

```diff
@@ -0,0 +1,131 @@
+import { FC } from 'react'
+import headerBG from '../../public/nav-bg.svg'
+import brandLogo from '../../public/logo_brand.svg'
+import useSupabaseAuth from '../hooks/useSupabaseAuth'
+import { Menu } from "@headlessui/react";
+import { version } from "../../package.json";
+import { capturePostHogAnayltics } from "../lib/analytics";
+import MobileNav from './MobileNav';
+
+const navs = [
+  {
+    placeholder: 'Docs',
+    links: "https://docs.opensauced.pizza/"
+  },
+  {
+    placeholder: 'Blog',
+    links: "https://dev.to/opensauced"
+  },
+  {
+    placeholder: 'Github',
+    links: "https://github.com/open-sauced"
+  },
+  {
+    placeholder: 'YouTube',
+    links: "https://www.youtube.com/opensauced"
+  }
+]
+
+const Nav:FC = () => {
+  const { signIn, signOut, user } = useSupabaseAuth();
+
+  return (
+    <div className='relative h-fit'>
+      <div className="absolute w-[100vw]  laptop:w-[120vw] left-[270px] mobile:hidden tablet:block tablet:w-[210vw] laptop:left-[638px] overflow-hidden z-[-1]">
+        <img src={headerBG} alt="" />
+      </div>
+      <div className='max-w-[950px] mx-auto px-[7px] '>
+        <div className='flex items-center justify-between'>
+            <div className='z-10 w-[172px] py-[10px] '>
+              <img src={brandLogo} alt="Open Sauced" className='w-full h-full' />
+            </div>
+
+            <MobileNav/>
+
+            <div className='z-10 py-[9px] flex mobile:hidden tablet:flex  gap-[50px] text-[14px] items-center text-white font-semibold leading-[20px] tracking-[-0.03em]'>
+              <ul className='flex gap-[50px]  '>
```

Comment:
Should those spaces be there ?

Reply:
just `npm run format` it will catch this